src/Tizen.NUI/src/public/Common/PropertyArray.cs

```diff
@@ -173,7 +173,7 @@ public PropertyArray Add(KeyValue value)
         public PropertyArray Add(PropertyValue value)
         {
             PropertyArray ret = new PropertyArray(Interop.Property.ArrayAdd(SwigCPtr, PropertyValue.getCPtr(value)), false);
-            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            //if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
```

Comment:
Could you check this? =>  `//if (... ;` 
if this is commented out, please remove it.
thank you.